src/PromiseRouter.js

```diff
@@ -39,6 +39,20 @@ PromiseRouter.prototype.route = function(method, path, handler) {
     throw 'cannot route method: ' + method;
   }
 
+  if (arguments.length > 3) {
```

Comment:
I'm not really a fan of this style of `arguments` length introspection. I find it is quite confusing, and kinda the opposite of self-documenting. Maybe doing some iteration in `getExpressRouter` would be a better way?